src/features/views/components/ViewColumnDialog/choices.tsx

```diff
@@ -69,6 +75,16 @@ const choices: ColumnChoice[] = [
       return <DoubleIconCardVisual color={color} icons={[Person, Person]} />;
     },
   },
+  {
+    key: CHOICES.TAG,
```

Comment:
Was this not added to the choices in your last PR? Or did this PR do changes to the tag choice object?

Reply:
Right! That was because in my PersonTag PR I didn't have your last changes, so I realised  doing the new issue that I needed to change that to stick with the new framework, hope it's okay!